### DISCUSSION TITLE
Infrastructure a new form

### DISCUSSION THREAD

**Discussion by beattobee** [community/community#66473] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

**Udacity's learning experience!**
open- ended, experiental projects are designed to reflect actual workplace challenges. They aren't just multiple choice questions or step- by- step guides, but instead require critical thinking.

1. Find answers to your questions with Knowledge, our proprietary wiki. Search questions asked by other students, connect with technical mentors, and discover how to solve the challenges that you encounter.
[Drone_Play_User_Manual.pdf](https://github.com/community/community/files/12558199/Drone_Play_User_Manual.pdf)

**Comment by mehulrajdev** [comment]
yes

**Comment by RoshdiRaed** [comment]
Response #1

**Reply by adamnoah03** [reply]
Good #1
